app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt

```diff
@@ -106,4 +108,11 @@ class MainActivity : AppCompatActivity() {
         mainActivityViewModel.updatePoolInfo()
         mainActivityViewModel.updateCompleteRelayState()
     }
+
+    private fun refreshSwipe(){
+        binding.swipeRefreshLayout.setOnRefreshListener {
+            updateData()
+            binding.swipeRefreshLayout.isRefreshing = false
+        }
```

Comment:
Let's move this at the end of `initView()` method.

Reply:
If you move this method to init View(), then it will not turn off the update indicator.